woocommerce/woocommerce
# Add config variable for simple product price to default.json

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds a new config variable into `tests/e2e/env/config/default.json` which allows setting the prince for the Simple Product created in the tests. Additionally, any tests that originally hard-coded `9.99` as the default price for the Simple Product have been updated to pull in the value from the config.

Closes https://github.com/woocommerce/woocommerce/issues/28574.

### How to test the changes in this Pull Request:

1. Make sure the e2e test suite passes locally and on Travis. The tests can be ran following [these instructions](https://github.com/woocommerce/woocommerce/tree/master/tests/e2e#running-tests). For example, the `Checkout page` tests (which now use the new variable) should pass as before:
![Screen Shot 2020-12-12 at 3 53 02 PM](https://user-images.githubusercontent.com/71906536/101998623-2b0b8880-3c92-11eb-92fa-239a60154f5d.png)
2. You can update the new price configure variable to another value and the tests still pass.
3. Test backwards compatibility: copy the `config.json` file over and remove the simple product price field. After doing so, run the tests. Make sure the tests still pass.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

N/A
